person.rb

```diff
@@ -21,6 +21,7 @@ def correct_name
 
   # define getter&setter for @name & @age
   attr_accessor :name, :age
+  attr_reader :rental
```

Comment:
- [ ] As written in the project requirements, the ` *Getters and Setters for @rentals should be present`. please make changes